code/renderers/vue3/src/__tests__/Button.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+defineProps<{ disabled: boolean; label: string }>();
+
+const emit = defineEmits<{
+    (e: 'myChangeEvent', id: number): void;
+    (e: 'myClickEvent', id: number): void;
+}>();
+
+</script>
+
+<template>
+    <button :disabled="disabled" @change="emit('myChangeEvent', 0)" @click="emit('myClickEvent', 0)">
```

Comment:
What's up with these tabs?

Reply:
Ah, no idea, maybe we haven't configured prettier with vue templates.

Reply:
Used prettier on it (my editor was somehow configured that volar was used for formatting)